docs/swagger.yaml

```diff
@@ -438,6 +484,47 @@ paths:
           description: User ID does not exist.
         500:
           description: Unexpected internal errors.
+  /users/{user_id}/password:
+    put:
+      summary: Change the password on a user that already exists.
+      description: |
+        This endpoint is for user to update password. Users with the admin 
+		role can change any user's password. Guest users can change only 
+		their own password.
+      parameters:
```

Comment:
There can not be multi-line of parameters that against the Swagger yaml specification.
You may refer this [http://swagger.io/specification/#parameterObject](http://swagger.io/specification/#parameterObject)